app/i18n/en-US/custom-loader.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Initiating doggo petting engine...": "Initiating doggo petting engine...",
+  "Compiling food pic library...": "Compiling food pic library...",
+  "Callibrating RNG blame threshold...": "Callibrating RNG blame threshold...",
+  "Dropping fire mixtape...": "Dropping fire mixtape...",
+  "Reticulating emoji splines...": "Reticulating emoji splines...",
+  "Waking up 24/7 support team...": "Waking up 24/7 support team...",
+  "Asking the stream gods for perfect settings...": "Asking the stream gods for perfect settings...",
+  "Powering up the clouds...": "Powering up the clouds...",
+  "Constructing additional widgets...": "Constructing additional widgets..."
```

Comment:
i don't think these jokes/references will translate perfectly to other languages, so i wonder if we can somehow add a note to these to add some localized references

Reply:
The jokes are definitely funny. But how about people who are stuck with them forever. I can't run Streamlabs OBS at all because it frozen on these funny comments. I even tried to uninstall the program and install again. Still doesn't work.

![Untitled-1](https://user-images.githubusercontent.com/30843221/71559489-2950b100-2a91-11ea-87b8-c2d944a65584.png)